src/main/java/com/pretz/dyntreeserver/service/dto/UserDTO.java

```diff
@@ -0,0 +1,56 @@
+package com.pretz.dyntreeserver.service.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class UserDTO {
+
+    private String name;
+
+    @JsonProperty(access = JsonProperty.Access.WRITE_ONLY)
+    private String password;
+
+    private String email;
+
+    public UserDTO() {
+    }
+
+    public UserDTO(String name, String password, String email) {
+        this.name = name;
+        this.password = password;
+        this.email = email;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    @JsonIgnore
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
+
+    public String getEmail() {
+        return email;
+    }
+
+    public void setEmail(String email) {
+        this.email = email;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
is it necessary? looks like default implementation (except for the lack of `password` field)

Reply:
I'm planning to use it in unit tests

Reply:
Code should not contain logic prepared specifically for unit tests :-)

Reply:
It's only toString, please don't be so papal